packaging/prerelease/build_app.sh

```diff
@@ -63,26 +63,34 @@ if [ ! "$nopull" = "1" ]; then
   "$client_dir/packaging/check_status_and_pull.sh" "$updater_dir"
 fi
 
+echo "Loading release tool"
+"$client_dir/packaging/goinstall.sh" "github.com/keybase/release"
+release_bin="$GOPATH/bin/release"
+
+echo "Wait for CI"
+"$release_bin" wait-ci --repo=client --commit=`git -C $client_dir log -1 --pretty=format:%h`
+"$release_bin" wait-ci --repo=kbfs --commit=`git -C $kbfs_dir log -1 --pretty=format:%h`
+
 if [ -n "$client_commit" ]; then
-  cd "$client_dir"
+  (cd "$client_dir"
   client_branch=`git rev-parse --abbrev-ref HEAD`
   function reset_client {
     (cd "$client_dir" && git checkout $client_branch)
   }
   trap reset_client EXIT
   echo "Checking out $client_commit on client"
-  git checkout "$client_commit"
+  git checkout "$client_commit")
```

Comment:
Could we indent the parenthesized blocks something like this?

```
(
    cd "$client_dir"
    client_branch=`git rev-parse --abbrev-ref HEAD`		    client_branch=`git rev-parse --abbrev-ref HEAD`
    function reset_client {		    function reset_client {
      (cd "$client_dir" && git checkout $client_branch)		      (cd "$client_dir" && git checkout $client_branch)
    }		    }
    trap reset_client EXIT		    trap reset_client EXIT
    echo "Checking out $client_commit on client"		    echo "Checking out $client_commit on client"
    git checkout "$client_commit"		 +  git checkout "$client_commit"
)
```

I worry someone will try to set an env var in there someday and get really confused otherwise.